contracts/eosio.system/src/rex.cpp

```diff
@@ -420,7 +420,7 @@ namespace eosiosystem {
    void system_contract::check_voting_requirement( const name& owner, const char* error_msg )const
    {
       auto vitr = _voters.find( owner.value );
-      check( vitr != _voters.end() && ( vitr->proxy || 21 <= vitr->producers.size() ), error_msg );
+      check( vitr != _voters.end() && ( vitr->proxy || 0 <= vitr->producers.size() ), error_msg );       // Required voters set to 0 
```

Comment:
Since `check_voting_requirement` is never called, I think we can leave this line alone, less changes is better for future maintenance.

Reply:
Also, line 201 is paying to 51 BPs, even though shares were calculated based on 42 max.